### DISCUSSION TITLE
プラグインのローカルインストールについて

### DISCUSSION THREAD

**Discussion by inosela** [growilabs/growi#8912] [Help / Questions] [answered]
こんにちは!　現場でGROWIを仮導入して好感触なため本番導入を考えています。

掲題について、設定からはプラグインインストーラーでgithubのリポジトリから持ってくるようになっていますが、社内ツールとして利用するためインターネットに出られない環境に構築しています。

その際のプラグインのインストール方法はございますか？　特に[growi-plugin-copy-code-to-clipboard](https://growi.org/plugins/weseek/growi-plugin-copy-code-to-clipboard)は手順書移行の際に便利なため、ぜひ導入をしたく。

**Comment by yuki-takei** [comment, marked as answer]
試用ありがとうございます

そうですね～想定されていたご相談ではあるのですが、現時点では github.com のみ受け付けるようになってしまっています。
https://github.com/weseek/growi/blob/5ff0aebca3df88bad64af69489c059435f3bc828/apps/app/src/features/growi-plugin/server/models/vo/github-url.ts#L60

以下のどちらかをやればなんとかなるけどどちらも面倒ではありますね…

1. 上記箇所を書き換えて起動
2. GROWI がインストールされているホストから github.com を lookup しに行った時の到達先をどこかに曲げて騙す

**Reply by inosela** [reply]
ご連絡ありがとうございます、現状把握いたしました。

cloneしたソースを社内のリモートリポジトリに登録してそれに向けるようにすれば何とか行ける感じですかね...

いただいた回答をもとに試してみます、ありがとうございました。
